share_app/src/main/java/org/odk/share/application/Share.java

```diff
@@ -28,7 +28,7 @@ public static Share getInstance() {
         return singleton;
     }
 
-    public static final String ODK_ROOT = Environment.getExternalStorageDirectory() + File.separator + "share";
+    public static final String ODK_ROOT = Environment.getExternalStorageDirectory() + File.separator + "odk";
```

Comment:
Changing this would move the share's database `share.db` from `/sdcard/share/` to `/sdcard/odk/`

Is this change really required for fixing this issue?

Reply:
@rkodev Do you agree with the above statement?

Reply:
@shobhitagarwal1612  yes, we can have this reverted back to the share directory